assets/javascripts/application.js

```diff
@@ -44,7 +44,7 @@ var attorneyBanner = require('./modules/attorneyBanner.js')
 var youtubePlayer = require('./modules/youtubePlayer.js')
 var accordion = require('./modules/accordion.js')
 var tabs = require('./modules/tabs.js')
-var charCounter = require('./modules/charCounter.js')
+var charCounter = require('../components/character-counter/character-counter.js')
```

Comment:
The other option is underscores. There are, of course, [arguments](https://superuser.com/questions/266491/using-underscore-in-file-names) [for](http://stackoverflow.com/questions/2740026/why-are-filename-underscores-better-than-hyphens) [either](https://blog.codinghorror.com/of-spaces-underscores-and-dashes/).